src/main/java/org/example/service/HelloService.java

```diff
@@ -0,0 +1,34 @@
+package org.example.service;
+
+
+public class HelloService {
+
+    private static HelloService INSTANCE;
```

Comment:
Check the naming conventions.
When the variable title should be written in caps.

Reply:
fixed